keyboards/opendeck/info.json

```
@@ -0,0 +1,4 @@
+{
+    "url": "https://github.com/cmdremily/OpenDeck",
+    "maintainer": "cmdremily",
```

```suggestion
    "maintainer": "cmdremily"
```
Should fix the info.json lint.